roseau/load_flow/io/dgs.py

```diff
@@ -116,7 +116,7 @@ def network_from_dgs(  # noqa: C901
                 logger.error(msg)
                 raise RoseauLoadFlowException(msg=msg, code=RoseauLoadFlowExceptionCode.DGS_BAD_PHASE_NUMBER)
 
-            line_types[type_id] = LineCharacteristics.from_sym(
+            line_types[type_id] = LineParameters.from_sym(
```

Comment:
You can go a bit further and make disappear every "type" word in variable names.
```python
line_parameters[parameter_id] = LineParameters.from_sym(...)
```